conf/application.conf

```diff
@@ -81,6 +81,11 @@ microservice {
       port = 9580
     }
 
+    pdf-generator-service {
```

Comment:
Is this added in app-config-base?

Reply:
https://github.com/hmrc/app-config-base/pull/1902